MKL_README.md

```diff
@@ -17,46 +17,3 @@ Installing and enabling the full MKL installation enables MKL support for all op
 
   5. Run 'sudo python setup.py install'
 
-# MKL2017 PLUGIN
```

Comment:
Shouldn't there be new documentation about MKLDNN instead?

Reply:
with new MKL-DNN integration, the installation has been tightly integrated (eg: mkl-dnn is a sub-module), no separate instructions are needed; the user just has to pass `USE_MKLDNN=1` or modify config.mk or cmake (just as other options).

Reply:
@marcoabreu @ashokei I'd like to create a new page (documentation) for MKL-DNN to introduce the installation, feature, performance, coverage, etc.
But we can make this page as-is now and I will back a new PR :)

Reply:
Okay, thanks @pengzhao-intel